<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>current</class>
 <widget class="QWidget" name="current">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>172</width>
    <height>307</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>172</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>172</width>
    <height>307</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>154</red>
        <green>151</green>
        <blue>146</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>154</red>
        <green>151</green>
        <blue>146</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>154</red>
        <green>151</green>
        <blue>146</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>154</red>
        <green>151</green>
        <blue>146</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frameEverything">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>172</width>
     <height>308</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>172</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>172</width>
     <height>308</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>161</red>
         <green>158</green>
         <blue>152</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>161</red>
         <green>158</green>
         <blue>152</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>161</red>
         <green>158</green>
         <blue>152</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>161</red>
         <green>158</green>
         <blue>152</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <widget class="QFrame" name="frameCurrent">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>142</y>
      <width>160</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="uiOpen">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>160</width>
       <height>75</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="uiName">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>20</y>
       <width>145</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: transparent; }</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiBirthDay">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>40</y>
       <width>73</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiDeathDay">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>58</y>
       <width>73</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: transparent; }</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiBirthPlace">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>40</y>
       <width>71</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: transparent; }</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiDeathPlace">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>58</y>
       <width>71</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: transparent; }</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiRelation">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>4</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: transparent; }</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameSpouse">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>222</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="uiOpenSpouse">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
    <widget class="QComboBox" name="uiSpouseList">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="uiMarrDay">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>26</y>
       <width>73</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: transparent; }</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiMarrPlace">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>26</y>
       <width>71</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: transparent; }</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiSpouseBirthDate">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>42</y>
       <width>73</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: transparent; }</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiSpouseBirthPlace">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>42</y>
       <width>71</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: transparent; }</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiSpouseDeathDate">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>58</y>
       <width>73</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: transparent; }</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiSpouseDeathPlace">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>58</y>
       <width>71</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: transparent; }</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="uiAddSpouse">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>2</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../qenes.qrc">
       <normaloff>:/icons/benefits.png</normaloff>:/icons/benefits.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="uiPicture">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>8</y>
      <width>157</width>
      <height>127</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../qenes.qrc"/>
 </resources>
 <connections/>
</ui>
